app/components/track-item/template.hbs

```diff
@@ -1,12 +1,39 @@
-{{#link-to 'channel.tracks.track.index' track class="Track-link"}}
-	<h3 class="Track-title">
-		{{track.title}}
-	</h3>
-	<p class="Track-body">{{track.body}}</p>
-{{/link-to}}
+{{#if inline}}
+	<a href="#" {{action "play" track}}>
```

Comment:
now we have action play and linkto. We used to have just actions, switched to links and now have both. I still ask for query param to be considered as I think they might be the best solution